docs/operate/get-started/basics.md

```diff
@@ -0,0 +1,99 @@
+---
+linkTitle: "Learn the basics"
+title: "Viam basics"
+weight: 10
+layout: "docs"
+type: "docs"
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+no_list: false
+description: "Learn the basics of Viam."
+carouselscript: true
+---
+
+## What is Viam?
+
+Viam is a software platform for data, AI, and automation.
+It drives your hardware (for example, cameras, sensors, and motors) and connects your devices to the cloud.
+
+Viam consists of:
+
+- An open-source binary called `viam-server` that runs on your machine, managing your hardware, software, and data.
+- A cloud app called the [Viam app](https://app.viam.com) that you can use to configure and manage your machines and data.
+- A registry of modules that support many different types of hardware and software.
+- Standardized APIs that abstract the underlying hardware and software.
+- SDKs for many programming languages that you can use to control your machines.
+
+`viam-server` connects your machine to the Viam app and to SDK clients.
+
+See [Viam architecture](/operate/reference/architecture/) for details on how Viam works.
+
+## What is a machine?
+
+A _machine_ is a computer (often a single-board computer like a Raspberry Pi or Jetson) or microcontroller and all the hardware attached to it, as well as the software running on it.
+You can think of one machine as representing one device, or one robot.
+Each machine runs an instance of `viam-server`.
+
+When you create a new machine in the Viam app, Viam generates a unique set of credentials for that machine that connect the physical machine to its instance in the Viam app.
+
+## What platforms does Viam run on?
+
+Viam can run on any computer that runs one of the following operating systems:
+
+- Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
+- macOS
+- Windows
+- 32-bit microcontrollers (limited support)
```

Comment:
```suggestion

Viam can also run on 32-bit microcontrollers such as the ESP32 series.
```

Suggesting this since above we say "computer" and "operating systems"

Reply:
Good point